docs/network-mobile/mobile-proof-of-coverage.mdx

```diff
@@ -37,6 +37,9 @@ un-tie a Radio's location from its signal strength and propagation.
 A CBRS Small Cell Radio ("Radio") can only operate as a transmitter; they cannot receive incoming
 data and thus can not "hear" one another directly.
 
+Unlike CBRS, Wi-Fi Access Points have the ability to "sniff" packet data of other wireless devices
+and networks to estimate where the Access Point is deployed.
```

Comment:
Thoughts on rephrasing this? 
"Sniff packet data" sounds alarming imo

Reply:
Just deleted it along with the section above.